air-dsl/Cargo.toml

```diff
@@ -10,3 +10,9 @@ codegen-winter = { package = "codegen-winter", path = "../codegen-winter", versi
 structopt = "0.3.26"
 env_logger = "0.9"
 log = { version = "0.4", default-features = false }
+winter-math = { package = "winter-math", version = "0.4", default-features = false }
+winter-utils = { package = "winter-utils", version = "0.4", default-features = false }
+winter-air = { package = "winter-air", version = "0.4", default-features = false }
```

Comment:
actually, one more note - are these imports actually required here? It doesn't seem like it, since we're not actually actually using the generated rust files which contain the imports that would require these dependencies

Reply:
I was using them to test for the other PR. Will remove them.